dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ErrorType.java

```diff
@@ -22,25 +22,35 @@
   CLASS_NOT_FOUND,
 
   /**
-   * The referenced class or interface found in the class path is not compatible with the source.
+   * The referenced class or interface found in the class path is not binary-compatible with the
+   * source class.
+   *
+   * <p>An example case of breaking binary-compatibility is when a superclass changes a method to
+   * {@code final} and a subclass is still overriding the method. Another example is when there is a
+   * method call to an interface and the interface is changed to a class with the same name.
+   *
+   * @see <a href="https://docs.oracle.com/javase/specs/jls/se7/html/jls-13.html#jls-13.4.9">Java
```

Comment:
Should all the JLS references be to the same version? (This one is Java 7, and the next is Java 8.)

Which version?

Reply:
Updated to Java 8.